src/nvim/path.c

```diff
@@ -1098,7 +1102,7 @@ int gen_expand_wildcards(int num_pat, char_u **pat, int *num_file,
     p = pat[i];
 
     if (vim_backtick(p))
-      add_pat = expand_backtick(&ga, p, flags);
+      add_pat = (int)expand_backtick(&ga, p, flags);
```

Comment:
The algorithm didn't change so, if before the type change it was guaranteed that the return value would fit into an `int` why wouldn't now? It cost me nothing to add an `assert` there but I don't think is necessary.